content/influxdb/v2.0/tools/grafana.md

```diff
@@ -81,11 +81,11 @@ configure your InfluxDB connection:
         ```
     - **Access**: Server (default)
 
-2. Under **Auth**, enable **Basic Auth**.
-3. Under **Basic Auth Details**, provide your InfluxDB authentication credentials:
+2. Under **Custom HTTP Headers**, select **Add Header**.
+3. Provide your InfluxDB authentication credentials:
 
-    - **User**: InfluxDB username
-    - **Password**: InfluxDB [authentication token](/influxdb/v2.0/security/tokens/)
+    - **Header**: "Authorization"
+    - **Value**: In the form of "Token <your InfluxDB [authentication token](/influxdb/v2.0/security/tokens/)>"
```

Comment:
- Combine `2.` and `3.` into a single line item
- Format "Authorization" as inline code
- With the head value, outline the pattern and provide an example:

~~~md
2. Under **Custom HTTP Headers**, select **Add Header**.
   Provide your InfluxDB authentication credentials:

    - **Header**: `Authorization`
    - **Value**: use the `Token` schema and provide your [InfluxDB authentication token](/influxdb/v2.0/security/tokens/) (for example: `Token y0uR5uP3rSecr3tT0k3n`)
~~~